stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl

```diff
@@ -5,7 +5,10 @@
 baremodule CompilerSupportLibraries_jll
 using Base, Libdl, Base.BinaryPlatforms
 
-export libgfortran, libstdcxx, libgomp, libatomic, libgcc_s
+export libgfortran, libstdcxx, libgomp, libgcc_s
+if !Sys.isfreebsd()
+    export libatomic
+end
```

Comment:
As I already said in https://github.com/JuliaLang/julia/issues/57808#issuecomment-2730962540, I'm confused: libatomic _is_ in https://github.com/JuliaBinaryWrappers/CompilerSupportLibraries_jll.jl/releases/download/CompilerSupportLibraries-v1.3.1%2B0/CompilerSupportLibraries.v1.3.1.x86_64-unknown-freebsd-libgfortran5.tar.gz.  I'm not convinced this the right solution, feels to me like libatomic is getting lost.

Reply:
Thanks for pointing that out @giordano , I think you're right.